src/types/select.js

```diff
@@ -4,12 +4,42 @@ import assign from "lodash.assign";
 import css from "./select.css";
 import multiple from "./lib/multiple.js";
 
+function findSelected(children) {
+    return children.find((option) => Boolean(option.selected));
```

Comment:
No argument there. I'd never seen anyone use `Boolean()` to cast before, I actually like it more than `!!` for precisely the reasons you mentioned.

Selected *should* be a Boolean, yes, haha. I was being overly cautious because this comes from the schema, which is user-generated, so I figured there was a just a dash of added clarity here that selected is a boolean attribute.